src/test/java/com/neo4j/docker/TestConfSettings.java

```diff
@@ -199,12 +198,53 @@ void testEnvVarsOverride() throws Exception {
         //Start the container
         container.setWaitStrategy(Wait.forHttp("/").forPort(7474).forStatusCode(200));
         container.start();
-        //Read the config file to check if the config is not overriden
+        //Read the debug.log to check that dbms.memory.pagecache.size was set correctly
         Stream<String> lines = Files.lines(logMount.resolve("debug.log"));
         Optional<String> heapSizeMatch = lines.filter(s -> s.contains("dbms.memory.pagecache.size=512m")).findFirst();
         lines.close();
         Assertions.assertTrue(heapSizeMatch.isPresent(), "dbms.memory.pagecache.size was not set correctly");
         //Kill the container
         container.stop();
     }
+
+    @Test
+    void testEnterpriseOnlyDefaultsConfigsAreSet () throws Exception {
+        Assumptions.assumeTrue(TestSettings.EDITION == TestSettings.Edition.ENTERPRISE,
+                "This is testing only ENTERPRISE EDITION configs");
+        //Create container
+        GenericContainer container = createContainer();
+        //Mount /logs
+        Path logMount = HostFileSystemOperations.createTempFolderAndMountAsVolume(container, "logs-", "/logs");
+        SetContainerUser.nonRootUser(container);
+        //Start the container
+        container.setWaitStrategy(Wait.forHttp("/").forPort(7474).forStatusCode(200));
+        container.start();
+        //Read debug.log to check that causal_clustering confs are set successfully
+        Stream<String> lines = Files.lines(logMount.resolve("debug.log"));
+        Optional<String> ccPresent = lines.filter(s -> s.contains("causal_clustering.transaction_advertised_address")).findFirst();
```

Comment:
I think it would be better if we actually tested that the configuration is also set to the value we expect. It looks as though entrypoint resolves `$(hostname)` to be the first 12 digits of the container id, so this works for me:

```
        String expectedTxAddress = container.getContainerId().substring( 0, 12 ) + ":6000";
        Stream<String> lines = Files.lines(logMount.resolve("debug.log"));
        Optional<String> ccPresent = lines.filter(s -> s.contains("causal_clustering.transaction_advertised_address="+expectedTxAddress)).findFirst();
        lines.close();
```